doc/articles/features/windows-ui-startscreen.md

```diff
@@ -15,15 +15,15 @@ The APIs are implemented on iOS and Android. Other platforms return `false` when
 
 `JumpListItem` supports `DisplayName`, `Description`, `Arguments`, and `Logo` properties on both Android and iOS.
 
-`DisplayName` may not be empty on Android and iOS - this behavior differs from UWP. If left empty, a single space will be used instead.
+`DisplayName` may not be empty on Android and iOS - this behavior differs from WinUI. If left empty, a single space will be used instead.
 
-`Logo` property can be initialized only with `ms-appx:`-based images. This behavior matches UWP and actually matches native support on both platforms as well.
+`Logo` property can be initialized only with `ms-appx:`-based images. This behavior matches WinUI and actually matches native support on both platforms as well.
 
-To handle `JumpListItem` activation, check the `LaunchActivatedEventArgs.Arguments` in `App.OnLaunched` method. Note that when the application is running, the method will still be called again (this behavior matches UWP).
+To handle `JumpListItem` activation, check the `LaunchActivatedEventArgs.Arguments` in `App.OnLaunched` method. Note that when the application is running, the method will still be called again (this behavior matches WinUI).
 
 The API supports interoperability with native "app shortcuts". This means items you add using native API instead of `JumpList` API will not be overwritten by `JumpList` and will appear first in the list of shortcuts. These non-Uno shortcuts will not be accessible in the `JumpList.Items` collection. To identify Uno-specific app shortcuts, a `UnoShortcut` key is used and set in `ShortcutInfo.Extras` on Android and `UIApplicationShortcutItem.UserInfo` on iOS.
```

Comment:
```suggestion
The API supports interoperability with native "app shortcuts". This means items you add using the native API instead of `JumpList` API will not be overwritten by `JumpList` and will appear first in the list of shortcuts. These non-Uno shortcuts will not be accessible in the `JumpList.Items` collection. To identify Uno-specific app shortcuts, a `UnoShortcut` key is used and set in `ShortcutInfo.Extras` on Android and `UIApplicationShortcutItem.UserInfo` on iOS.
```